SmartDeviceLink-iOS/SmartDeviceLink/SDLV2ProtocolMessage.m

```diff
@@ -21,8 +21,9 @@ - (instancetype)initWithHeader:(SDLProtocolHeader *)header andPayload:(NSData *)
 
 // Convert RPC payload to dictionary (for consumption by RPC layer)
 - (NSDictionary *)rpcDictionary {
+    
     // Only applicable to RPCs
-    if (self.header.serviceType != SDLServiceType_RPC) {
+    if (self.header.serviceType != SDLServiceType_RPC && self.header.serviceType != SDLServiceType_BulkData) {
```

Comment:
Please change to `(self.header.serviceType != SDLServiceType_RPC) && (self.header.serviceType != SDLServiceType_BulkData)` so that there isn't a merge conflict with #360